roles/ceph-facts/tasks/facts.yml

```diff
@@ -104,22 +104,22 @@
     # this task shouldn't run in a rolling_update situation
     # because it blindly picks a mon, which may be down because
     # of the rolling update
-    - name: is ceph running already?
-      command: "{{ timeout_command }} {{ _container_exec_cmd | default('') }} ceph --cluster {{ cluster }} -s -f json"
+    - name: get current fsid if cluster is already running
+      command: "{{ timeout_command }} {{ _container_exec_cmd | default('') }} ceph --cluster {{ cluster }} fsid"
```

Comment:
you forgot to add `-f json` otherwise the `fsid` key won't be present in the json output